sdk/src/client/api/high_level.rs

```diff
@@ -121,14 +121,20 @@ impl Client {
 
         let network_info = self.get_network_info().await?;
 
-        if let Some(latest_ms_timestamp) = network_info.latest_milestone_timestamp {
+        if let Some(latest_finalized_slot_timestamp) = network_info.latest_finalized_slot.map(|idx| {
+            idx.as_timestamp(
+                network_info.protocol_parameters.genesis_unix_timestamp(),
+                network_info.protocol_parameters.slot_duration_in_seconds(),
+            )
+        }) {
             // Check the local time is in the range of +-5 minutes of the node to prevent locking funds by accident
-            if !(latest_ms_timestamp - FIVE_MINUTES_IN_SECONDS..latest_ms_timestamp + FIVE_MINUTES_IN_SECONDS)
+            if !(latest_finalized_slot_timestamp - FIVE_MINUTES_IN_SECONDS
```

Comment:
Finalized slot is already a bit older than the current time, anything against using `relativeAcceptedTangleTime`?

Reply:
What is that?

Reply:
One of the info response fields: https://github.com/iotaledger/tips-draft/blob/43b25b17b564bc42963d9f32645a98f1bc2a58c3/tips/TIPS-API/openapi3-core.yaml#L2505

Reply:
done

Reply:
Please also rename `latest_finalized_slot_timestamp` here then

Reply:
oop